docs/downloads/automation-library/integrations/askAI/askAI_CR.cm

```diff
@@ -15,8 +15,8 @@ automations:
         args:
           comment: |
             # ✨ gitStream Review ✨
-            {{ source | askAI("
-              Review the PR code diff.
+            {{ source | askAI("Code reviewer",
+              "Review the PR code diff.
               - Identify bugs, security risks, and performance issues
               - Check for deprecated methods and style guide violations
               - Provide specific improvement suggestions based on the changes", env.OPEN_AI_TOKEN) | encode }}
```

Comment:
does this multline string works with the Nunjucks engine? asking to make sure it was tested, including playground

Reply:
Tested in this repo (https://github.com/linear-b/gitstream/blob/main/.cm/ask-ai.cm)